website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -177,6 +177,6 @@ The project can now be built, and the operator can be deployed on-cluster. For f
 [memcached_cr]: https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml
 [memcached_types]: https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/pkg/apis/cache/v1alpha1/memcached_types.go
 [kb_memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/master/example/memcached-operator/memcached_controller.go.tmpl
-[kb_quickstart]: /docs/building-operators/golang/quickstart/
+[kb_tutorial]: /docs/building-operators/golang/tutorial/
```

Comment:
There is another place in the [`overview`](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/overview/_overview.md) page which points towards the golang tutorial. That needs to be updated too, I suppose ?

Reply:
I updated there as well. It will change in the pr https://github.com/operator-framework/operator-sdk/pull/3543 too. Really tks